### ISSUE TITLE
Question: Linting inline scripts which use custom script-tags

### ISSUE THREAD

**Issue by seiz** [jshint/jshint#2014]
Hi,

for use in our own CMS-Product, we have kind of a custom templating language which embeds javascript inside custom script tags when used in .html pages.

We're using `<$` instead of `<script>` and `$>`instead of `</script>`. Is there any configuration option where i could pass a regex to jshint, so it'd support my custom script tag and properly find the javascript inside our custom templates?

Here's a brief example of how one of our .html template-files might look (`<$= $>` is a special syntax, which values writes to output):

```
<$
    var customerName = "Fred Feuerstein";
$>

<h1>Hello <$= customerName $></h1>
<p>How are you today?</p>
```

Thanks a lot for any pointers!
Stefan

**Comment by seiz** [comment]
Excuse my (very) late reply. Sorry, i don't know what PI tags are and DuckDuck didn't turn out anything useful  in that regard either.

Anyways, so currently, there still is no way, i can make jshint scan scripts inside the mentioned tags in the example above, right? No workaround either?

**Comment by jugglinmike** [comment]
You are correct: JSHint does not offer that functionality. There have been no
comments on this issue for three years, and this indicates that few users are
interested in such a feature. That's an important requirement both to motivate
a contributor to implement new functionality and to convince the maintainers
that its worth the ongoing maintenance cost it will have.

I can offer a workaround, though: pre-process your templates (according to the
transformation you described in the original report) immediately prior to
running JSHint. You can avoid the overhead of writing a temporary file by using
the command line interfaces `--filename` option.

    $ cat my-file.not-js | my-transformation-script.sh | jshint --filename my-file.not-js -

Note that trailing hyphen (`-`) character! It's what causes the CLI to read
from the standard input stream.

Hope that helps!
